Mu2eUtilities/src/TriggerResultsNavigator.cc

```diff
@@ -53,8 +60,8 @@ namespace mu2e {
     for  (unsigned int i=0; i< _trigPathsNames.size(); ++i){
       std::string name     = _trigPathsNames[i];
       size_t      index    = findTrigPath(name);
-      bool        accepted = accept(name);
-      printf("// %24s  %2li       %i    //\n", name.c_str(), index, accepted == true ? 1:0);
+      bool        good     = accepted(name);
+      printf("// %24s  %2li       %i    //\n", name.c_str(), index, good == true ? 1:0);
```

Comment:
I would strongly suggest to leave out printf and use cout (or Mu2e message streamer)